### DISCUSSION TITLE
Using `outdir` name starting with dot doesn't seem to work

### DISCUSSION THREAD

**Discussion by moaali** [chakra-ui/panda#3072] [Q&A] [answered]
### Description

When using _dot_ as the starting character for `outdir` name (e.g. `.panda`) files don't get generated despite logged as generated.

![Screenshot 2024-12-31 at 1 56 40 PM](https://github.com/user-attachments/assets/81ba0594-47b7-4781-b65e-be75374d8ec0)

### Link to Reproduction

https://stackblitz.com/edit/vitejs-vite-ge2r6zsu?file=panda.config.ts

### Panda CSS Version

latest

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
It works as expected on my machine. This is probably happening because Stackblitz hides folders that start with `.*`

Try it locally, and it should work.

![CleanShot 2024-12-31 at 15 22 19@2x](https://github.com/user-attachments/assets/f3a19527-43b8-4faf-b26f-48d5c2694cca)
